frontend/src/index.html

```diff
@@ -79,6 +79,98 @@
         </div>
       </div>
     </nav>
+
+    <!-- player UI design -->
+    <div class="my-player text-test fixed z-10 bottom-0 w-full bg-white border border-solid border-gray-300 rounded-t-md shadow-upward">
```

Comment:
Avoid using semantic CSS classes like `my-player` here and below. We are using classes only for styling. Where needed, use `id` attribute to query element from JS.